### PR TITLE
Changes to dispute info on markets info

### PR DESCRIPTION
**Pull request by JohnDanz**
https://github.com/AugurProject/augur/issues/3255

updated disputeInfo object to remove invalid, add dispute round, and added preFilledStake to stake details

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). ✨

 This link will always point to this PR's latest changes: https://peek.run/qo0hj7m

File: packages/augur-sdk/src/state/getter/Markets.ts [hunk 1]
```diff
@@ -161,9 +162,9 @@ export interface DisputeInfo {
 
 export interface StakeDetails {
   outcome: string;
-  isInvalid: boolean;
```
Comment (adrake33) [inline packages/augur-sdk/src/state/getter/Markets.ts:164]: Good call, we don't really need this anymore.

File: packages/augur-tools/src/flash/scripts.ts [hunk 2]
```diff
@@ -420,7 +420,7 @@ export function addScripts(flash: FlashSession) {
       let preEmptiveStake = '0';
       if (extraStake) {
         preEmptiveStake = new BigNumber(extraStake)
-          .dividedBy(QUINTILLION)
+          .multipliedBy(QUINTILLION)
```
Comment (adrake33) [inline packages/augur-tools/src/flash/scripts.ts:423]: Cool, looks like you found the cause of that bug you were seeing!

Comment (bthaile) [inline packages/augur-tools/src/flash/scripts.ts:423]: 1 REP === 1 * 10 ** 18 attoREP, correct?

Comment (JohnDanz) [reply packages/augur-tools/src/flash/scripts.ts:423]: yeah

**Review by adrake33 — APPROVED** [review]
Lgtm

File: packages/augur-tools/src/flash/scripts.ts (hunk 2 above)
Comment (bthaile) [reply packages/augur-tools/src/flash/scripts.ts:423]: yup, I made the mistake. I gotcha!

Comment (bthaile) [reply packages/augur-tools/src/flash/scripts.ts:423]: yup, goog fix

**Pull request merged**